Gemfile

```diff
@@ -24,6 +24,7 @@ gem 'sass-rails', '~> 6.0.0'
 gem 'sendgrid', '~> 1.2.4'
 gem 'sprockets', '~> 3.7.2'
 gem 'yaaf', '~> 2.2'
+gem 'wdm', '>= 0.1.0' if Gem.win_platform?
```

Comment:
looks like a gem used for development, only on local, is that right? if that's the case, then maybe is worth to move it to the `group :development do` (line 44). what do you think?